### PR TITLE
LTD-5261-add-licence-status-details-tab 

### PR DESCRIPTION
**Pull request by depsiatwal**
https://uktrade.atlassian.net/browse/LTD-5261

This is to display Licence Status Details. 

Currently we are displaying the last Licence object that was created. Having looked at real data  this appears to reflect the latest status as the first one is normally Cancelled state. 

API PR 
https://github.com/uktrade/lite-api/pull/2126

### CODE REVIEW

File: caseworker/templates/case/slices/summary.html [hunk 1]
```diff
@@ -229,6 +229,20 @@
...
+		<dt class="app-case__summary-list__key">
+			Licence Status
+		</dt>
+		<dd class="app-case__summary-list__value">
+			<span class="govuk-hint govuk-!-margin-0">
+				{% if licence_status %}
+					{{ licence_status}}
+				{% else %}
+					No Licence
+				{% endif %}
```
Comment (markj0hnst0n) [inline caseworker/templates/case/slices/summary.html:242]: The comments on the ticket suggest this should be set to 'No licence status set'

File: unit_tests/caseworker/cases/views/test_case_details.py [hunk 2]
```diff
@@ -289,6 +289,38 @@ def test_case_details_sub_status_change_displayed(
...
 
 
+@pytest.mark.parametrize(
+    "licence_data, expected_status, expected_text",
+    (
+        ([], None, "No Licence"),
+        ([{"created_at": "2023-08-08", "status": "Issued"}], "Issued", "Issued"),
+        (
+            [{"created_at": "2023-08-08", "status": "Revoked"}, {"created_at": "2023-08-09", "status": "Revoked"}],
+            "Revoked",
```
[the remark this answers is not in the document]
Comment (depsiatwal) [reply unit_tests/caseworker/cases/views/test_case_details.py:299]: ahh that's what it was suppose to be updated

File: caseworker/templates/case/slices/summary.html (hunk 1 above)
Comment (depsiatwal) [reply caseworker/templates/case/slices/summary.html:242]: updated

**Review by markj0hnst0n — APPROVED** [review]
looks grand

File: caseworker/cases/helpers/licence.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+def get_latest_licence_status(case):
+    licenses = case.get("licences")
+    if licenses:
+        sorted_licenses = sorted(licenses, key=lambda x: x["created_at"])
+        return sorted_licenses[-1].get("status")
```
Comment (saruniitr) [inline caseworker/cases/helpers/licence.py:5]: could this be refactored as below?
```
if not licences:
    return None

sorted_licenses = sorted(licenses, key=lambda x: x["created_at"])
return sorted_licenses[-1].get("status")

**Review by saruniitr — APPROVED** [review]
one suggestion which I leave it with you

File: caseworker/cases/helpers/licence.py (hunk 3 above)
Comment (depsiatwal) [reply caseworker/cases/helpers/licence.py]: done

**Pull request merged**